source/ios/AdaptiveCards/AdaptiveCards/AdaptiveCards/ACOIResourceResolver.h

```diff
@@ -0,0 +1,22 @@
+//
+//  ACOIResourceResolver.h
+//  ACOIResourceResolver
+//
+//  Copyright © 2018 Microsoft. All rights reserved.
+//
+
+#import <UIKit/UIKit.h>
+
+@protocol ACOIResourceResolver
+
+- (UIImage *)resolveImageResource:(NSString *)url;
```

Comment:
>NSString * [](start = 35, length = 10)

Why not make this an NSURL, if it's supposed to be a URL?

Reply:
that's very good point

---
In reply to: [202427902](https://github.com/Microsoft/AdaptiveCards/pull/1702#discussion_r202427902) [](ancestors = 202427902)